docker/openemr/flex-3.11/utilities/devtools.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh
+
+
+if [ $1 == "psr2-report" ]; then
+    docker exec -i $(docker ps | grep _openemr | cut -f 1 -d " ") sh -c 'cd openemr; php -d memory_limit=640M /root/.composer/vendor/squizlabs/php_codesniffer/bin/phpcs -n --extensions=php,inc --standard=ci/phpcs.xml --report=full .'
```

Comment:
note these commands will be running from inside the docker, so this one will be:
```
cd /var/www/localhost/htdocs/openemr
php -d memory_limit=640M /root/.composer/vendor/squizlabs/php_codesniffer/bin/phpcs -n --extensions=php,inc --standard=ci/phpcs.xml --report=full .
```

Reply:
I'd also echo something before each one like "Running PSR2 report" or something like that